### DISCUSSION TITLE
DAI-2024-2025 - My Picocli project - Maxime Lestiboudois

### DISCUSSION THREAD

**Discussion by MaximeLesti** [heig-vd-dai-course/heig-vd-dai-course#341] [Show and tell]
https://github.com/MaximeLesti/java_intellij-idea-and-maven

**Comment by ludelafo** [comment]
Everything seems fine for me except the `pom.xml`. It seems you have removed the initial properties from the following lines:

```xml
<properties>
    <!-- Omitted for brevity -->
</properties>
```

You must keep the initial values to ensure Maven can encode the files correctly (more on this in the next course) and check for the right Java version.
